src/openboardview/PDFBridge/PDFBridgeSumatra.cpp

```diff
@@ -309,7 +341,12 @@ void PDFBridgeSumatra::DocumentSearch(const std::string &str, bool wholeWordsOnl
 	}
 
 	auto wstr = utf8_to_utf16(strCopy);
-	std::wstring ddeCmd = L"[Search(\"" + this->pdfPathString + L"\",\"" + wstr + L"\"," + (caseSensitive ? L"1" : L"0") + L")]";
+	std::wstring ddeCmd;
+	this->TestFork();
+	if (this->IsForked)
+		ddeCmd = L"[Search(\"" + this->pdfPathString + L"\",\"" + wstr + L"\"," + (caseSensitive ? L"1" : L"0") + L")]";
+	else
+		ddeCmd = L"[Search(\"" + this->pdfPathString + L"\",\"" + wstr + L"\")]";
```

Comment:
This could be simplified to `if (this->TestFork());` if the above suggestion is taken. Not sure how I feel about having the assignment nested in an `if..else` vs having an additional ternary in the assignment... I think split over two or three lines it's still fairly readable?

```suggestion
	std::wstring ddeCmd = L"[Search(\"" + this->pdfPathString + L"\",\"" + wstr + L"\""
		+ (this->TestFork() ? L"," + (caseSensitive ? L"1" : L"0") : L"") + L")]";
```